timebot/timebot.go

```diff
@@ -3,10 +3,24 @@
 package timebot
 
 import (
+	"fmt"
 	"time"
+	"strings"
 )
 
 // ParseTime takes a string and returns time.Time in time.UTC
 func ParseTime(text string) (time.Time, bool) {
-	return time.Time{}, true
+	const longForm = "2006-01-02 15:04 -0700"
```

Comment:
```suggestion
	const longForm = "2006-01-02 15:04 MST"
```

이게 제가 처음에 생각했던 것이었거든요.
이러면 PST/PDT 는 자동으로 해결되고 KST만 커스텀으로 하면 되거든요.

지금 하신 방법도 굉장히 새롭네요 하지만 PST/PDT 직접 관리하는게 까다롭기 때문에 
형식이 두개로 하는 방법은 어떠한가요?

```golang
forms := []string{"2006-01-02 15:04 MST", "2006-01-02 15:04 -0700"}

for _, f := range forms {
  t, err := time.Parse(f, text)
  if err == nil {
    return t, true
  }
}

return t, false
```

Reply:
제가 꼼꼼히 챙겨볼 시간이 안되지만 
1. 만드신 테스트만 볼때는 KST 가 테스트 통과되고 PST가 통과되지 않아 -0700 을 이용했습니다.
2. 구글링해보니 Go에서 숫자형식을 지향한다는 코멘트를 읽었습니다. 그래서 일괄적으로 모두 숫자로 변경하는 테이블을 하나 두고 사용하는게 어떨까 생각했습니다. - 그런데 이거 쉬운문제가 아니군요. PST 나 MST 는 같은 이름 다른 타임존도 있군요. 어떤게 정답일까 고민이됩니다.

Reply:
앗 MST는 단순 파서 포맷이고 같지 않습니다 2006년이랑 같은 파서 포맷입니다. 

제가 얘기한건 미국은 여름에 섬머타임을 사용합니다.
PDT가 있고 PST가 있습니다. 지금은 PST이고 PDT가 아니지요. 

일단 이건 머지하고 다음 테스트 케이스를 추가하겠습니다.

Reply:
아래와 같은 의미로 코멘트 한 것입니다.
MST | Malaysia Standard Time | UTC+08
MST | Mountain Standard Time (North America) | UTC−07

PST | Pacific Standard Time (North America) | UTC−08
PST | Philippine Standard Time | UTC+08

Reply:
ㅋㅋㅋㅋ 이럴수가
일단 확실히 테스트케이스를 먼저 추가해야겠네요